frontend/src/pages/UserProfile.jsx

```diff
@@ -169,12 +170,12 @@ function UserProfile(props) {
                         </ul>
 
                         <div className="card-title">
-                            <a
+                            <Link
```

Comment:
proba they work the same way. Does the original <a> tag not work for you?

Reply:
yeah when using an <a> tag it threw an error, im pretty sure you have to use <Link> for internal redirects